stock_valued_picking_report/__openerp__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014 Pedro M. Baeza - Tecnativa <pedro.baeza@tecnativa.com>
+# Copyright 2015 Antonio Espinosa - Tecnativa <antonio.espinosa@tecnativa.com>
+# Copyright 2016 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Valued picking list",
+    "version": "9.0.1.0.0",
+    "author": "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    "website": "www.tecnativa.com",
+    "category": "Warehouse Management",
+    "license": "GPL-3",
+    "depends": [
+        "base",
```

Comment:
No need to add `base` unless it's the only one.

OTOH I miss here the `decimal_precission` module.